src/sidebar/tabcenter.css

```diff
@@ -577,13 +582,24 @@ body[platform="win"] #searchbox-input:placeholder-shown {
 .tab-icon {
     width: 16px;
     height: 16px;
+    background-image: url("img/defaultFavicon.svg");
     background-repeat: no-repeat;
     background-position: center;
     background-size: contain;
     background-origin: content-box;
     image-rendering: pixelated;
 }
 
+.tab-icon.fill-color {
+    -moz-context-properties: fill;
+    fill: var(--icon-color);
+    filter: url("img/filters.svg#fill");
+}
+
+#tablist:not(.shrinked) .tab-icon.fill-color {
+    fill: #5a5b5c;
```

Comment:
Do we need a different color in these circumstances?

Reply:
I've also noticed that this triggers a bug where the colors are different:

<img width="137" alt="screen shot 2018-05-21 at 12 47 04 pm" src="https://user-images.githubusercontent.com/6424575/40319150-1dd8bc20-5cf5-11e8-88a2-f4691f02cf4e.png">